repository.py

```diff
@@ -939,7 +940,7 @@ def listPackagesFromRepos(repos, rpm_pattern, query_format='%{nevr}'):
         repo._accessor.start()
 
     try:
-        createMainYumConfig(yum_conf_path, repos, cachedir)
+        createMainYumConfig(yum_conf_path, repos, cachedir, override_repogpgcheck=0)
```

Comment:
Aren't we also disabling GPG checks to install the packages with that line?

Reply:
replying to myself: no, because of the other commit in the PR

Reply:
well, even without renaming for config file, the installer should be regenerating the yum config before each operation, so that ought to be safe anyway